easybuild/tools/job/gc3pie.py

```diff
@@ -249,25 +252,25 @@ def complete(self):
             self._engine.progress()
 
             # report progress
-            self._print_status_report(['total', 'NEW', 'SUBMITTED', 'RUNNING', 'ok', 'failed'])
+            self._print_status_report()
 
             # Wait a few seconds...
             time.sleep(self.poll_interval)
 
         # final status report
-        self._print_status_report(['total', 'ok', 'failed'])
+        print_msg("Done processing jobs", log=self.log)
+        self._print_status_report()
 
     @gc3pie_imported
-    def _print_status_report(self, states=('total', 'ok', 'failed')):
+    def _print_status_report(self):
         """
         Print a job status report to STDOUT and the log file.
 
-        The number of jobs in any of the given states is reported; the
+        The number of jobs in each states is reported; the
         figures are extracted from the `stats()` method of the
-        currently-running GC3Pie engine.  Additional keyword arguments
-        can override specific stats; this is used, e.g., to correctly
-        report the number of total jobs right from the start.
+        currently-running GC3Pie engine.
         """
         stats = self._engine.stats(only=Application)
-        job_overview = ', '.join(["%d %s" % (stats[s], s.lower()) for s in states if stats[s]])
-        print_msg("GC3Pie job overview: %s" % job_overview, log=_log, silent=build_option('silent'))
+        states = ', '.join(["%d %s" % (stats[s], s.lower()) for s in stats if s != 'total' and stats[s]])
+        total = len(self.jobs)
```

Comment:
I can't see how this yields the correct nr. of jobs -- it should be `len(self.jobs._deps)` IMHO